datalad/support/sshconnector.py

```diff
@@ -604,6 +796,11 @@ def close(self, allow_fail=True, ctrl_path=None):
             self._connections = dict()
 
 
+# retain backward compat with 0.13.4 and earlier
```

Comment:
Why not to move that check of cfg from `__init__` here and bind those two accordingly on the `cfg` setting right here?

Reply:
I don't know.

Reply:
It could be stripped also. I don't know if any consumer.